test/new-e2e/tests/sysprobe-functional/apmtags_test.go

```diff
@@ -295,6 +295,9 @@ func (v *apmvmSuite) TestUSMAutoTaggingSuite() {
 	testExe := path.Join("c:", "users", "administrator", "littleget.exe")
 	vm.CopyFile("usmtest/littleget.exe", testExe)
 
+	pipeExe := path.Join("c:", "users", "administrator", "NamedPipeCmd.exe")
```

Comment:
What does the binary file `NamedPipeCmd.exe` include?

Reply:
Is it the binary for `tools/NamedPipeCmd/main.go`?

Reply:
Yes.  It is a CLI tool to hit the named pipe in tests.  There was a comment in the previous PR about building the tool from source, but I actually strongly disagree, because it would be quite common to hit cyclic build dependency breaks.